.github/workflows/docs.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           deploy_key: ${{ secrets.ACTIONS_DOCUMENTATION_DEPLOY_KEY }}
           publish_branch: master
           publish_dir: ./docs/build/html
-          external_repository: bluesky/blueskyproject.io
+          external_repository: bluesky/bluesky.github.io
```

Comment:
Actually, the plan was to remove this entire action, and let the one below publish to the gh-pages branch of this repo. Then we turn on gh-pages hosting for this repo, and https://blueskyproject.io/even-model should start hosting the docs from this repo, rather than from the `bluesky.github.io` repo. I didn't spot that both of these actions made it into skeleton PR...